medical_practitioner/views/medical_menu.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    Copyright 2017 Creu Blanca
+    Copyright 2017 Eficent Business and IT Consulting Services, S.L.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <data>
+
+        <menuitem id="medical_conf_practitioner_menu"
```

Comment:
As this module deprecates Physician, we are replacing this menu https://github.com/LasLabs/vertical-medical/blob/feature/10.0/medical_physician/medical_physician/views/medical_menu.xml#L20 for that new one.

Maybe I don't see your point, but IMO a new menu is necessary.

All the other changes are already added. Thanks!